test/unit/mdc-icon-button/foundation.test.js

```diff
@@ -56,6 +56,20 @@ test('#constructor sets on to true if the toggle is pressed', () => {
   assert.isTrue(foundation.isOn());
 });
 
+test('#handleClick calls #toggle', () => {
+  const {foundation, mockAdapter} = setupTest();
+  foundation.init();
+  foundation.handleClick();
+  td.verify(mockAdapter.setAttr('aria-pressed', 'true'), {times: 1});
```

Comment:
I think this should verify that `foundation.toggle()` was called. The aria-pressed is verified in another test.